api/managementpb/dbaas/components.proto

```diff
@@ -9,6 +9,15 @@ import "google/api/annotations.proto";
 
 // TODO move it to parent directory / package once API is v1-stable
 
+// DatabaseType contains databases supported by DBaaS.
+enum DatabaseType {
+  DATABASE_TYPE_INVALID = 0;
```

Comment:
We don't actually need that. What we need is enum with operators. Because we already use operator_type and each operator has one and only database type.

Reply:
TODO CHANGE THIS